projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/sample/SampleRegistrationRequest.java

```diff
@@ -17,25 +16,30 @@
  * @param replicateReference  the biological replicated reference the sample has been taken from
  * @param sampleOrigin        information about the sample origin.
  * @param analysisType        analysis to be performed
+ * @param analysisMethod      analysis method to be permed
  * @param comment             comment relating to the sample
  * @since 1.0.0
  */
 public record SampleRegistrationRequest(String label, BatchId assignedBatch,
                                         ExperimentId experimentId, Long experimentalGroupId,
                                         BiologicalReplicateId replicateReference,
-                                        SampleOrigin sampleOrigin, String analysisType, String comment) {
+                                        SampleOrigin sampleOrigin, String analysisType,
+                                        AnalysisMethod analysisMethod, String comment) {
 
   public SampleRegistrationRequest(String label, BatchId assignedBatch, ExperimentId experimentId,
       Long experimentalGroupId, BiologicalReplicateId replicateReference,
-      SampleOrigin sampleOrigin, String analysisType, String comment) {
+      SampleOrigin sampleOrigin, String analysisType, AnalysisMethod analysisMethod,
+      String comment) {
     this.label = Objects.requireNonNull(label);
     this.assignedBatch = Objects.requireNonNull(assignedBatch);
     this.experimentId = Objects.requireNonNull(experimentId);
     this.experimentalGroupId = Objects.requireNonNull(experimentalGroupId);
     this.replicateReference = Objects.requireNonNull(replicateReference);
     this.sampleOrigin = Objects.requireNonNull(sampleOrigin);
-    this.comment = comment;
     this.analysisType = analysisType;
+    this.analysisMethod = Objects.requireNonNull(analysisMethod);
```

Comment:
```suggestion
    this.analysisMethod = Objects.requireNonNull(analysisMethod, "analysisMethod must not be null");
```
It is always good to also provide a message here. Depending on the java runtime and how exceptions are propagated, this can make debugging simpler.